packages/connect/src/popup.ts

```diff
@@ -96,7 +100,9 @@ export const setupListener = <T>({
   // Send a message to the authenticator popup at a consistent interval. This allows
   // the authenticator to 'respond'.
   const pingInterval = 250;
-  const interval = setInterval(() => {
+  let interval: number | undefined = undefined;
```

Comment:
Could tidy this up as a recursive promise

Reply:
Ah, this code needs to go :+1: